server/dhcpgui.conf

```diff
@@ -43,6 +43,9 @@ ip_address_lease_time = 300
 #   the broadcast address could also be 192.168.137.255
 broadcast_address = '255.255.255.255'
 
+# Listen to DHCP requests on this interface
+bind_address = '192.168.137.10'
```

Comment:
Hi, the rest looks ready to merge. Could you put in the default value here so it uses all addresses when people start it first?
I think, it is good to have this example.
Does 192.168.137.0 also work?

Reply:
The default that will work on all addresses is '' (blank). This will cause the DHCP server to listen on all IP addresses. I'll do this. I'll put a non-blank example in the comments.

Given your subnet mask, I don't think 192.168.137.0 will work. I avoid x.y.z.0 for this reason. Check out:
https://serverfault.com/questions/10985/is-x-y-z-0-a-valid-ip-address

Reply:
Added the works-everywhere value. Please take a look.